### ISSUE TITLE
[Android]emoji cannot be shown in Android platform where weex version is 0.20.0.2

### ISSUE THREAD

**Issue by zjutkz** [apache/incubator-weex#2432]
## Bug description
Emoji cannot be shown in Android platform.

## Environment
* Device: OnePlus 6t
* OS: Android 9.0
* Android SDK Version: 0.20.0.2
* JS Framework Version: 0.29.0
* Vue Version: 2.5.16-weex.5

#Source code
`<template>
  <text>
      这是一个emoji😄啦啦啦啦
  </text>
</template>`

## Screenshots
![10081557800159_ pic_hd](https://user-images.githubusercontent.com/9446934/57666095-5307b580-7631-11e9-87ec-024de16155db.jpg)

## Additional context
It was perfectly shown when I used Android SDK Version 0.18.0, but failed in 0.20.0.2

**Comment by YorkShen** [comment]
Please attach the link of demo in [weex editor](http://editor.weex.io/), then reopen the issue.

**Comment by zjutkz** [comment]
@YorkShen I tried to run my code in Android playground,It works fine when I compiled the weex_sdk module `compile project(':weex_sdk')`, but when I dependent 0.20.0.2 in build.gradle `compile 'com.taobao.android:weex_sdk:0.20.0.2'`, emoji cannot be shown.
p.s. I found a [pr](https://github.com/apache/incubator-weex/pull/1804/commits/12bc5aedd27342646a507aff91a0ff9272467e98), is this pr fix the problem?
